Kernel/Bus/VirtIO/Console.cpp

```diff
@@ -14,7 +14,11 @@ unsigned Console::next_device_id = 0;
 
 UNMAP_AFTER_INIT RefPtr<Console> Console::try_create(PCI::Address address)
 {
-    auto console = adopt_ref_if_nonnull(new Console(address));
+    auto control_receive_buffer = Memory::RingBuffer::try_create("VirtIOConsole control receive queue", CONTROL_BUFFER_SIZE);
+    auto control_transmit_buffer = Memory::RingBuffer::try_create("VirtIOConsole control transmit queue", CONTROL_BUFFER_SIZE);
+    if ((!control_receive_buffer.is_error()) || (control_transmit_buffer.is_error()))
```

Comment:
Accidental `!`? (Also there's no need to wrap each `is_error` check in an extra set of parentheses